extensions/consul/deployment/pom.xml

```diff
@@ -46,9 +46,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-jackson-deployment</artifactId>
         </dependency>
+        <!-- TODO revert back to camel-quarkus-core-deployment when upgrading to camel 3.2 -->
```

Comment:
An issue wold be nice so that we do not forget

Reply:
#1040